code/datums/elements/ore_collecting.dm

```diff
@@ -0,0 +1,26 @@
+/*
+ * A component to allow us to collect ore
+ */
+/datum/element/ore_collecting
+
+
+/datum/element/ore_collecting/Attach(datum/target)
+	. = ..()
+
+	if(!isliving(target))
+		return ELEMENT_INCOMPATIBLE
+	RegisterSignal(target, COMSIG_HOSTILE_PRE_ATTACKINGTARGET, PROC_REF(collect_ore))
+
+/datum/element/ore_collecting/Detach(datum/target)
+	. = ..()
+	UnregisterSignal(target, COMSIG_HOSTILE_PRE_ATTACKINGTARGET)
+
+/datum/element/ore_collecting/proc/collect_ore(mob/living/source, atom/target)
+	SIGNAL_HANDLER
+
```

Comment:
Bit silly but just for consistency maybe check for combat mode in case someone wants to bludgeon the box with some ore.
Or would that mean that mooks are always doing it?

Reply:
this element is just for picking up ore from the ground